components/image.css

```diff
@@ -0,0 +1,20 @@
+@import url(../styles.css);
+
+.image-responsive{
+    padding: 2rem;
+}
+
+.image-responsive img{
+    height: 100%;
+    width: 100%;
+}
+
+.image-circular{
+    padding: 2rem;
+}
+
+.image-circular img{
+    border-radius: 50%;
+    height: 100%;
+    width: 100%;
```

Comment:
It is not coming as circular, you should try aspect-ratio: 1/1 CSS property.

Reply:
Thanks, Naman I will Work on it.

Reply:
instead of that can I use here height and width the same?